distributed/comm/ucx.py

```diff
@@ -61,6 +109,13 @@ def cuda_array(n):
                     "In order to send/recv CUDA arrays, Numba or RMM is required"
                 )
 
+    pool_size_str = dask.config.get("rmm.pool-size")
+    if pool_size_str is not None:
+        pool_size = parse_bytes(pool_size_str)
+        rmm.reinitialize(
+            pool_allocator=True, managed_memory=False, initial_pool_size=pool_size
```

Comment:
@pentschev would you like to see options for `pool_allocator` and `managed_memory` as well ?

Reply:
I think for the time being we can skip that.